Complete overhaul of connected lights tutorial (#41)
ARMmbed/connected-lights-cloud: docs/7_updates.md

Run:

```
$ mbed dm update device -D YOUR_ENDPOINT_NAME
```

Replace `YOUR_ENDPOINT_NAME` with the endpoint name in Device Management.

> Inspect the logs on the device (via a serial monitor) to see the firmware update progress (this may take a few minutes). It looks similar to:

Reviewer: ```suggestion
Inspect the logs on the device (via a serial monitor) to see the firmware update progress. (This may take a few minutes.) It looks similar to:
```

```
Mbed Bootloader
[DBG ] Update active firmware
[DBG ] Erase active application
[DBG ] Write header
[DBG ] Copy application
[DBG ] Verify application
[DBG ] New active firmware is valid
booting...
```

When the download completes, the firmware is verified. If everything is OK, the firmware update is applied. Your device is now running the latest version of the application.

> You can also verify that the firmware update was successful via the "Events Log" tab on the device page in the Device Management portal.

Reviewer: ```suggestion
You can also verify the firmware update was successful through the **Events Log** tab on the device page in the Device Management portal.
```
